### PR TITLE
move types to src

### PR DESCRIPTION
**Pull request by szy21**
## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: src/model_getters.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-function moisture_model(parsed_args)
+function get_moisture_model(parsed_args)
```
Comment (jiahe23) [inline src/model_getters.jl:1]: I like this new name!

**Comment by jiahe23** [comment]
can we give `example_types` a better name? Or can we maybe move functions in this file to other places based on their functionalities, e.g., `get_atmos` to `model_getters.jl` and `get_space` to `common_spaces.jl`

**Comment by szy21** [comment]
> can we give `example_types` a better name? Or can we maybe move functions in this file to other places based on their functionalities, e.g., `get_atmos` to `model_getters.jl` and `get_space` to `common_spaces.jl`

As discussed offline, this will be done as a separate PR.

**Pull request merged**